kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/GcpClientBuilder.java

```diff
@@ -41,7 +41,7 @@
 public abstract class GcpClientBuilder<Client> {
 
   public enum KeySource {
-    FILE, JSON
+    FILE, JSON, SERVICE_ACCOUNT
```

Comment:
Nit: Do you think `SERVICE_ACCOUNT` may be a little misleading? I understand that that's the use case you'd like to satisfy here, but the docs for [`GoogleCredentials::getApplicationDefault`](https://googleapis.dev/java/google-auth-library/0.21.0/com/google/auth/oauth2/GoogleCredentials.html#getApplicationDefault--) outline several other places that credentials will be picked up from if we use that method:

> 1. Credentials file pointed to by the GOOGLE_APPLICATION_CREDENTIALS environment variable
> 2. Credentials provided by the Google Cloud SDK gcloud auth application-default login command
> 3. Google App Engine built-in credentials
> 4. Google Cloud Shell built-in credentials
> 5. Google Compute Engine built-in credentials

Additionally, we originally used the keyfile method because that was how were able to generate and provide service account credentials to the connector; it may confuse people who want to use a service account's keyfile with the connector since they'd probably see `SERVICE_ACCOUNT` and think that that's the option they'd want to use.

Do you think something like `APPLICATION_DEFAULT` or `ADC` may be better here?